customer-node/service/ticketsService.js

```diff
@@ -0,0 +1,19 @@
+const {
+  findTicketsByCustomer, createTicket, findTicketById, deleteTicket,
+} = require('../data_access/ticketda');
+
+const getTicketsByCustomerId = (customerId) => findTicketsByCustomer(customerId);
+
+const addTicket = (ticket) => createTicket(ticket);
+
+const cancelTicket = async (customerId, ticketId) => {
+  const ticket = await findTicketById(ticketId);
+  if (ticket[0].customerId !== customerId) {
```

Comment:
Check if ticket is null before doing this comparison

Reply:
done